README.md

```diff
@@ -43,7 +43,30 @@ it, you will have to run `cmake` manually like so:
   ```bash
   cmake -DCMAKE_INSTALL_PREFIX=/path/of/install/dir
  ```
- 
+
+The target of the complex_bessel library is exported
+as `complex_bessel::complex_bessel` to a package configuration file for this
+library.
+
+After installation you can find this library in your
+project CMakeLists.txt with:
+  ```cmake
+  find_package(complex_bessel)
+```
+or you can just put a copy of `complex_bessel` source code with zero changes
+into your source tree and just add it from the upper level CMakeLists.txt
+  ```cmake
+ add_subdirectory(complex_bessel)
+```
+To compile and link this library you should have C\C++14 and Fortran compiles
```

Comment:
typo: C++14 and Fortran compilers*

Reply:
It seems that you have to use pure C for linking (if I remove C from cmake project description the linking fails)

Reply:
Sorry, C was only for linkage with hdf5. Updated.